src/App.js

```diff
@@ -9,6 +10,7 @@ function App() {
       <Router>
         <Switch>
           <Route path="/" exact component={Home} />
+          <Route path="/" exact component={Favorites} />
```

Comment:
Both `Route`s are linking to the same URL. So basically you are rendering both components at the same time. I would suggest a sub-page for your `Favorites` component:
```suggestion
          <Route path="/" exact component={Home} />
          <Route path="/favorites" exact component={Favorites} />
```